client/client.go

```diff
@@ -716,17 +716,16 @@ func (c *Client) Download(name string, dest Destination) (err error) {
 // Target returns the target metadata for a specific target if it
 // exists. If it does not, ErrNotFound will be returned.
 func (c *Client) Target(name string) (data.TargetFileMeta, error) {
-	m, err := c.Targets()
-	if err != nil {
-		return data.TargetFileMeta{}, err
+	target, err := c.getTargetFileMeta(util.NormalizeTarget(name))
```

Comment:
Ok, I see, this is interesting. So because [getTargetFileMeta()](https://github.com/theupdateframework/go-tuf/blob/014a8782ee18aeca37536142f195dd5b43a44bfd/client/delegations.go#L12)->[loadLocalSnapshot()](https://github.com/theupdateframework/go-tuf/blob/014a8782ee18aeca37536142f195dd5b43a44bfd/client/delegations.go#L60)->[getLocalMeta()](https://github.com/theupdateframework/go-tuf/blob/014a8782ee18aeca37536142f195dd5b43a44bfd/client/client.go#L278), we have the same effect as calling [Targets()](https://github.com/theupdateframework/go-tuf/blob/014a8782ee18aeca37536142f195dd5b43a44bfd/client/client.go#L733) previously, which also called getLocalMeta(), correct?

Reply:
Exactly